vitess.io/_includes/footer.html

```diff
@@ -1,15 +1,8 @@
 <footer role="contentinfo" id="site-footer">
   <nav role="navigation" class="menu bottom-menu">
-    <ul class="menu-item">
-      {% for link in site.data.footer %}
-      {% if link.url contains 'http' %}
-        {% assign domain = '' %}
-        {% else %}
-        {% assign domain = site.url %}
-      {% endif %}
-      <li><a href="{{ domain }}{{ link.url }}" {% if link.url contains 'http' %}target="_blank"{% endif %}>{{ link.title }}</a></li>
-      {% endfor %}
-    </ul>
+    {% assign separator = '&nbsp;&nbsp;<b>·</b>&nbsp;&nbsp;' %}
+    <a href="https://groups.google.com/forum/#!forum/vitess" target="_blank">Contact: vitess@googlegroups.com</a>{{ separator }}
+    <a href="https://groups.google.com/forum/#!forum/vitess-announce" target="_blank">Announcements</a>{{ separator }}
+    &#169; {{ site.time | date: '%Y' }} <a href="{{ site.url }}">{{ site.title }}</a> powered by <a href="http://www.google.com">Google Inc</a>
```

Comment:
On your preview instance, the "Vitess" link turns into a link to whatever current page you're on, instead of linking back to the home page. Can we instead just hard-code it like this:

```
<a href="/">Vitess</a> powered by...
```